section1/reflection.md

```diff
@@ -1,19 +1,64 @@
 ## Section 1 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
-
+```
+-When asking questions, it depends on the area of study. I tend to wait somewhere in between.
+-I am very independent and will exhaust all resources before reaching out.
+```
 2. What are the data types you learned about in this section? In your own words, define each.
+-The date types I learned in this section are the following:
+  ### JavaScript Primitive data types-
+```
+- String -consists of a letter and other character all wrapped w/ quotes or '' can be used. 
+Note: open and closing quote ,=must be consistent.
 
-3. How would you log the string `"Hello World!"` to the console?
+- Number -all numbers including negative -, decimal 0.25, and large #'s no comma is used. ex: 3456
 
+- Boolean -True or False that triggers a path
+```
+3. How would you log the string `"Hello World!"` to the console?
+```
+console.log( "Hello World!")
+```
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
 
+```
+- To indicate a comment in JavaScript use double back slash preceding a comment.
+- A code comment relays imperative information in written and spoken language.
+   -This helps communicate what is needed when collaborating or explain why an alteration was made.
+```
+
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
 
+```
+- A variable is like a nickname that has a complex different meaning. 
+- In code you can substitute a nickname(variable) for a lot of information.
+  - EX: a nick name of "za" means "pizza"
+```
+
 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
 
+```
+I frequently use Gmail.
+  - Three variables would be: name, directories, files.
+```
+
 7. In your own words, explain what concatenation is.
 
+```
+Concatenation: Means "join together." Join together strings using the (+). Just like in adding #'s together. 
+In context it does something different
+```
+
 8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
+```
+- Developers use concatenation in Gmail when they join cells together.
+- When they need to join an email function like "star" and that connects to the "star folder.
+```
 
 9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+
+```
+-In Atom what code do we use to create spaces in the Terminal output?
```

Comment:
You can add a string of a space `" "` or the `/n` will return to the next line in the terminal output.